src/Parser/BalanceDruid/Modules/Spells/FullMoon.js

```diff
@@ -15,7 +15,17 @@ class FullMoon extends Module {
   firstMoonCast = false;
   orderFound = false;
 
-  fmAvailableCasts = 0;
+  getfmAvailableCasts() {
```

Comment:
`get fmAvailableCasts()` will allow you to use this as a property (this.fmAvailableCasts)